nebula_hw_interfaces/include/nebula_hw_interfaces/nebula_hw_interfaces_hesai/hesai_cmd_response.hpp

```diff
@@ -21,6 +21,7 @@
 
 #include <array>
 #include <cstdint>
+#include <iomanip>
```

Comment:
Why is this needed?

Reply:
`std::setw` and `std::setfill` further down